### ISSUE TITLE
bug inputNumber thousand separete

### ISSUE THREAD

**Issue by sourdoth** [primefaces/primeng#9713]
**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
inputNumber not add dot when number is <9999 only add dot for >10.000 (thousand separete ".")

**Expected behavior**
add thousand separete "." when number > 999
example: 1000 -> 1.000

**Minimal reproduction of the problem with instructions**

section "Integer Only" 
https://www.primefaces.org/primeng/showcase/#/inputnumber

* **Angular version:** 11

* **PrimeNG version:** 11

**Comment by pmo1948** [comment]
![image](https://user-images.githubusercontent.com/69136429/104475592-f6a82680-55c7-11eb-982a-9ccfe5fb83e3.png)

I was not able to reproduce the bug...

**Comment by sourdoth** [comment]
I have set i18N locale 'es' may be is that

![imagen_2021-01-14_160524](https://user-images.githubusercontent.com/29101756/104636582-53681780-5682-11eb-8467-757266f745f0.png)

**Comment by rsanhueza-indra** [comment]
I have the same problem when I use the 'es-CL' locale:

`<p-inputNumber locale="es-CL"></p-inputNumber>`

**Comment by mertsincan** [comment]
I think this issue has been fixed in 12.1.1.

For now, closed. If the problem persists, please reopen it.
Best Regards,
